argo-cd-apps/base/monitoring-workload-in-cluster.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: monitoring-workload-in-cluster
+
+spec:
+  project: workload-monitoring
+
+  source:
+    path: components/monitoring/base
+    repoURL: https://github.com/redhat-appstudio/infra-deployments.git
+    targetRevision: main
+
+  destination:
+    namespace: workload-monitoring
```

Comment:
are you sure about the namespace name? why not `openshift-user-workload-monitoring`?